docs/literate/src/files/adding_new_parabolic_terms.jl

```diff
@@ -18,8 +18,15 @@ equations_hyperbolic = LinearScalarAdvectionEquation2D(advection_velocity);
 # `ConstantAnisotropicDiffusion2D` has a field for `equations_hyperbolic`. It is useful to have
 # information about the hyperbolic system available to the parabolic part so that we can reuse
 # functions defined for hyperbolic equations (such as `varnames`).
-
-struct ConstantAnisotropicDiffusion2D{E, T} <: Trixi.AbstractEquationsParabolic{2, 1}
+#
+# The abstract type `Trixi.AbstractEquationsParabolic` has three parameters: `NDIMS` (the spatial dimension,
+# e.g., 1D, 2D, or 3D), `NVARS` (the number of variables), and `GradientVariable`, which we set as
+# `GradientVariablesConservative`. This indicates that the gradient should be taken with respect to the
+# conservative variables (e.g., the same variables used in `equations_hyperbolic`). Users can also take
+# the gradient with respect to a different set of variables; see, for example, the implementation of
+# [`CompressibleNavierStokesDiffusion2D`](@ref), which can utilize either "primitive" or "entropy" variables.
+
+struct ConstantAnisotropicDiffusion2D{E, T} <: Trixi.AbstractEquationsParabolic{2, 1, GradientVariablesConservative}
```

Comment:
Can we find a way to make this choice automatic? Otherwise, it will be a breaking change.

Reply:
As far as I know, no, since I think it would require the ability to set "default" parameters. https://discourse.julialang.org/t/type-parameters-by-default/45577

I can hold off on this PR until we are ready for a breaking release. The stuff that would require this is an optimization, and doesn't prevent me from implementing other features.